uiowa/uiowa
# Feature: Gray background link color updates

Resolves https://github.com/uiowa/uiowa/issues/4537. 

This is the pull request to integrate https://github.com/uiowa/uids/pull/650.

# How to test

`blt frontend`
`blt ds --site=sandbox.uiowa.edu`
- Go to https://sandbox.uiowa.ddev.site/text-area-link-color and compare against https://sandbox.prod.drupal.uiowa.edu/text-area-link-color and verify that links are blue if a white or gray background is blue. 
  - Headings will **not** be blue, neither will the links within the gold `<thead>` on tables.
